omnibus/package-scripts/agent-deb/prerm

```diff
@@ -153,6 +153,18 @@ remove_apm_remote_config()
     fi
 }
 
+remove_auth_token()
+{
+    # Since Agent 7.56, Agent stores its token file in /opt/datadog-agent/run/auth_token.
+    # This is a best-effort solution, as users can
+    # decide to put this file in another place by changing the top-level
+    # run_path config value.
+    if [ -f "$INSTALL_DIR/run/auth_token" ]; then
+        echo "Removing auth_token file"
+        rm "$INSTALL_DIR/run/auth_token" || true
+    fi
```

Comment:
I currently do not see anything being cleaned in `/etc/datadog-agent` and indeed after deleting the Agent package `/etc/datadog-agent/auth_token` and auto-generated `/etc/datadog-agent/datadog.yaml` are still there.

I think it makes sense to remove`/etc/datadog-agent/auth_token`, just wondering about global cleanup of `/etc/datadog-agent`.

Reply:
Config files are usually handled differently by package managers, for DEB they are kept until the the package is purged, for RPM I think they are renamed to `.rpmold` 
It would make sense to force the removal for the auth token since the package managers most likely don't know about it and won't attempt to delete it

Reply:
Added it